src/containers/pages/subjectinformation/SubjectQuickSearch.js

```diff
@@ -115,7 +116,7 @@ class SubjectQuickSearch extends Component<Props, State> {
       [SUBJECT_INFORMATION.RACE]: person.getIn([PERSON_RACE_FQN, 0], ''),
       [SUBJECT_INFORMATION.GENDER]: person.getIn([PERSON_SEX_FQN, 0], ''),
       [SUBJECT_INFORMATION.AGE]: moment().diff(moment(person.getIn([PERSON_DOB_FQN, 0], '')), 'years'),
-      [SUBJECT_INFORMATION.SSN_LAST_4]: 'XXXX',
+      [SUBJECT_INFORMATION.SSN_LAST_4]: person.getIn([PERSON_SSN_LAST_4_FQN, 0], ''),
```

Comment:
If we're obfuscating the actual last 4 of the SSN when loading existing people, it seems sufficient to just make that box disabled, so we don't actually need this

Reply:
I've added type="password" to this field so the digits will be replaced by the browser's native password symbol.